src/Providers/Stripe/Customer/Service.php

```diff
@@ -33,7 +33,7 @@ public function create(Token $token, string $email, string $description): Custom
                 $params['payment_method'] = $token->value;
 
                 break;
-            case StripeToken::CARD:
+            case StripeToken::TOKEN:
```

Comment:
is there a reason why this was renamed?

Reply:
Yeah I wasn't using the right token type the CARD token is never going to be used in that case.